src/Compilers/CSharp/Test/Symbol/Symbols/MissingSpecialMember.cs

```diff
@@ -1588,11 +1588,9 @@ public struct S
 
             var compilation = CreateCompilationWithMscorlib45(source);
             compilation.MakeMemberMissing(SpecialMember.System_Nullable_T_GetValueOrDefault);
-            compilation.VerifyEmitDiagnostics(
-                // (23,19): error CS0656: Missing compiler required member 'System.Nullable`1.GetValueOrDefault'
-                //             S.v = s ?? -1;
-                Diagnostic(ErrorCode.ERR_MissingPredefinedMember, "s").WithArguments("System.Nullable`1", "GetValueOrDefault").WithLocation(23, 19)
-                );
+
+            // We use more optimal `GetValueOrDefault(defaultValue)` member for this case, so no error about missing `GetValueOrDefault()` is reported
+            compilation.VerifyEmitDiagnostics();
```

Comment:
Seems like we should add a version of this test that marks both members missing.

Reply:
I tried to keep all tests for this optimization close together in `CodeGenOperators.cs`. The test you are requesting here is already there and is named `TestNullCoalesce_NullableNonDefaultConstant_MissingGetValueOrDefaultAndGetValueOrDefaultWithADefaultValueParameter`
This test was fixed afterwards and I don't see much value in bringing other 'missing member' test cases to this file